src/controllers/restful/PaymentController.ts

```diff
@@ -18,37 +19,57 @@ import { PaymentsCheckResponse } from "../../types/controllers/PaymentsCheckResp
 import { CodiceContestoPagamento } from "../../types/PagoPaTypes";
 import * as PaymentsConverter from "../../utils/PaymentsConverter";
 import * as RestfulUtils from "../../utils/RestfulUtils";
-// Forward a payment check request from BackendApp to PagoPa
+
+/**
+ * This controller will be invoked by BackendApp.
+ * It's necessary to retrieve information about a qrcode (payment)
+ * It will require payment information to PagoPa using VerificaRPT service
+ * @param {express.Request} req - The RESTful request
+ * @param {express.Response} res - The RESTful response to fill with payment information
```

Comment:
I don't understand why this function express.Response gets an `express.Response` object for sending the response to the client AND returns a promise with `PaymentsCheckResponse`? it's either one or the other

Reply:
the suggested approach is to just return a promise, see how [this controller](https://github.com/teamdigitale/italia-backend/blob/master/src/controllers/messagesController.ts#L21) has been implemented

Reply:
I'm doing a little refactor to show whet @cloudify is suggesting here, we will elaborate on it together @LsKoder